[BUG]: Error when running DFP benchmarks

### Version

24.03

### Which installation method(s) does this occur on?

Conda

### Describe the bug.

Getting `AssertionError: log_level must be specified` when running DFP benchmarks following instructions [here](https://github.com/nv-morpheus/Morpheus/blob/branch-24.03/examples/digital_fingerprinting/production/morpheus/benchmarks/README.md).

### Minimum reproducible example

```shell
pytest -s --benchmark-enable --benchmark-warmup=on --benchmark-warmup-iterations=1 --benchmark-autosave benchmarks/test_bench_e2e_dfp_pipeline.py::test_dfp_stages_duo_training_e2e
```


### Relevant log output

```
>           assert log_level is not None, "log_level must be specified"
E           AssertionError: log_level must be specified

../../../../morpheus/utils/logger.py:186: AssertionError
======================================================================================= short test summary info =======================================================================================
FAILED benchmarks/test_bench_e2e_dfp_pipeline.py::test_dfp_stages_duo_training_e2e - AssertionError: log_level must be specified
========================================================================================== 1 failed in 1.09s ==========================================================================================

```


### Full env printout

_No response_

### Other/Misc.

_No response_

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

--- a/examples/digital_fingerprinting/production/morpheus/benchmarks/test_bench_e2e_dfp_pipeline.py
+++ b/examples/digital_fingerprinting/production/morpheus/benchmarks/test_bench_e2e_dfp_pipeline.py
@@ -105,7 +105,7 @@ def dfp_training_pipeline_stages(pipe_config: Config,
                                  filenames: typing.List[str],
                                  reuse_cache=False):
 
-    configure_logging(logger.level)
+    configure_logging(log_level=logger.level)
 
     pipeline = LinearPipeline(pipe_config)
     pipeline.set_source(MultiFileSource(pipe_config, filenames=filenames))
@@ -157,7 +157,7 @@ def dfp_inference_pipeline_stages(pipe_config: Config,
                                   output_filepath: str,
                                   reuse_cache=False):
 
-    configure_logging(logger.level)
+    configure_logging(log_level=logger.level)
 
     pipeline = LinearPipeline(pipe_config)
     pipeline.set_source(MultiFileSource(pipe_config, filenames=filenames))

